library/Jira/Web/Form/NewIssueForm.php

```diff
@@ -62,16 +62,36 @@ protected function assemble()
             return;
         }
 
-        $projects = $this->jira->get(sprintf(
-            'issue/createmeta?projectKeys=%s',
-            rawurlencode($projectName)
-        ))->getResult()->projects;
+        $serverInfo = $this->jira->get('serverInfo')->getResult();
```

Comment:
Please add this to `RestApi`.
Introduce `getVersion()` and for example `isServer()`method.